codec-http2/src/main/resources/META-INF/native-image/io.netty/codec-http2/native-image.properties

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 The Netty Project
+#
+# The Netty Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+Args = --rerun-class-initialization-at-runtime=io.netty.handler.codec.http2.Http2CodecUtil \
```

Comment:
What is the reason to use `--rerun-class-initialization-at-runtime` as opposed to `--delay-class-initialization-to-runtime?

Reply:
In old versions of graal we couldn't delay this class as it was needed by other initializers, I just double checked and with rc16 this seems not to be the case anymore, we can just add the class to the `delay` list and remove the `rerun`.

Reply:
Thanks, I have done it in: https://github.com/netty/netty/pull/9118